src/Symfony/Component/HttpKernel/DataCollector/RouterDataCollector.php

```diff
@@ -18,6 +18,8 @@
 
 /**
  * @author Fabien Potencier <fabien@symfony.com>
+ *
+ * @internal since Symfony 4.4
```

Comment:
you meant `@final` I suppose

Reply:
Nope it's extended in FWB so we can't make it `@final`.

Reply:
I see. Maybe just mark `collect` as final in this class? This would be enough wouldn't it?

Reply:
That works indeed since the method is not overidden.